src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -887,7 +887,7 @@ The Chebyshev center is the center of a largest Euclidean ball enclosed by `P`.
 In general, the center of such a ball is not unique (but the radius is).
 """
 function chebyshev_center(P::AbstractPolyhedron{N};
-                          get_radius::Bool=false,
+                          compute_radius::Bool=false,
```

Comment:
Looking at the implementation, the radius is always computed. I think it's cleaner to define `chebyshev_center` and `chebyshev_radius`, 

```julia
function chebyshev_center(...)
    c, _ = _chebyshev(P)
end
function chebyshev_radius(...)
    _, r = _chebyshev(P)
end
``` 
The function `underapproximate` can call `_chebyshev` directly.

Reply:
Or otherwise make `chebyshev_center` always return center and radius tuple.

Reply:
I agree and will make the proposed change.

Reply:
I went for renaming to `chebyshev_center_radius` for clarity. We currently do not need `chebyshev_center` and `chebyshev_radius`, but this way we can easily add them later.

There was also a bug in the implementation for `Interval`: It returned the radius 0.

Reply:
I rebased with `master`. The new changes are in the last commit.